common/src/KokkosKernels_BlockUtils.hpp

```diff
@@ -52,13 +52,13 @@ KOKKOS_INLINE_FUNCTION void kk_block_add(const size_type block_dim, value_type *
 // Note: block is assumed to be row-major, dense matrix (no extra padding)
 // Note: set clear=true to set C = 0 before increment
 template <typename size_type, typename value_type,
-          typename DGEMM = KokkosBatched::SerialGemmInternal<KokkosBatched::Algo::Gemm::Unblocked>>
+          typename DGEMM = KokkosBatched::Impl::SerialGemmInternal<KokkosBatched::Algo::Gemm::Unblocked>>
 KOKKOS_INLINE_FUNCTION void kk_block_dgemm(const size_type block_dim, value_type *dst, const value_type *valA,
                                            const value_type *valB, const bool clear = false) {
   const auto ZERO = static_cast<value_type>(0);
   const auto ONE  = static_cast<value_type>(1);
-  DGEMM::invoke(block_dim, block_dim, block_dim, ONE, valA, block_dim, 1, valB, block_dim, 1, clear ? ZERO : ONE, dst,
-                block_dim, 1);
+  DGEMM::invoke(KokkosBlas::Impl::OpID(), KokkosBlas::Impl::OpID(), block_dim, block_dim, block_dim, ONE, valA,
```

Comment:
I'm confused about how calling `OpID()` here with no arguments works. It looks like it must take a value.

Reply:
Thank you for the comments. 
I am doing exactly the same thing as in gemv. Basically, I am just passing a temporary object of `OpID()` class.

https://github.com/kokkos/kokkos-kernels/blob/85bbf1ff4aba86881c6bd65e8b00ec2fd21d6e9e/blas/impl/KokkosBlas2_serial_gemv_impl.hpp#L91-L107

Reply:
To be clear, in that spot Yuuichi is calling the constructor not `operator()` so that should work fine.